src/lib/actions/RuleDescribeAction.ts

```diff
@@ -87,9 +84,18 @@ export class RuleDescribeAction implements Action {
 		});
 	}
 
-	private displayStyledRule(rule: DescribeStyledRule, jsonEnabled: boolean): void {
-		// TODO: We should remove this instantiation of new Ux in favor of possibly a new method on Display
-		new Ux({jsonEnabled: jsonEnabled})
-			.styledObject(rule, ['name', 'engine', 'runWith', 'isPilot', 'enabled', 'categories', 'rulesets', 'languages', 'description', 'message']);
+	private displayStyledRule(rule: DescribeStyledRule): void {
```

Comment:
Handling this TODO right now because the testability is impossible without having this data tracked on our own Display object.

I confirmed that this did not break rule describe or rule describe --json in any way. Still outputting the same information and all of our other tests around this still pass.